pkg/addons/addons_gcpauth.go

```diff
@@ -64,6 +65,20 @@ func enableOrDisableGCPAuth(cfg *config.ClusterConfig, name string, val string)
 }
 
 func enableAddonGCPAuth(cfg *config.ClusterConfig) error {
+	// Disable the gcloud update check if the env var isn't already set
+	if _, c := os.LookupEnv(disableCheckEnvVar); !c {
+		err := os.Setenv(disableCheckEnvVar, "1")
```

Comment:
this is global, lets set this pass this env as part of the Command Exec Runner so only passed to the commands that we run as opposed to globally